### PR TITLE
Fresh Sentry projects

### PR DESCRIPTION
**Pull request by lmath**
### Why do we need this? 
We scrubbed PII here:  https://github.com/guardian/support-frontend/pull/494
As we stop sending PII, to sentry, we are also deleting old sentry projects and starting anew. 

Both support and support-client-side shall point to new projects in conjunction with this PR. 

### The changes 
* Pointing to the DSN of the new project for support-client-side
* Update the DSN configured in the config for the new sentry project for support
* Move to using sentry-logback (raven-logback is deprecated)
* Wrap Sentry.init in a try 

TODO:
Pre-merge
- [x] Test in CODE that error logs go to the new project
- [x] Add the DSN for the new sentry project to the PROD config 
- [ ] Remove the DSN from the CODE config

Post-merge
- [ ] Delete the existing sentry project
- [ ] Rename the new sentry projects back to support and support-client-side as before

### trello card/screenshot/json/related PRs etc
[**Trello Card**](https://trello.com/c/LiojyKYR/1555-delete-all-projects-which-contain-pii-from-sentry-and-replace-with-a-clean-project)
[Scrub PII from logs sent to sentry]( https://github.com/guardian/support-frontend/pull/494)

### CODE REVIEW

File: app/monitoring/SentryLogging.scala [hunk 1]
```diff
@@ -1,31 +1,28 @@
...
+import scala.util.{Failure, Success, Try}
+
+object SentryLogging {
+  def init(config: Configuration) {
+    config.sentryDsn match {
+      case None => SafeLogger.warn("No Sentry logging configured (OK for dev)")
+      case Some(sentryDSN) =>
+        SafeLogger.info(s"Initialising Sentry logging with $sentryDSN")
+        Try {
+          val sentryClient = Sentry.init(sentryDSN)
```
Comment (lmath) [inline app/monitoring/SentryLogging.scala]: We were setting a release on the SentryAppender, but we're sending this information to Sentry as a tag anyway, so is this this needed? Also, I'm not covering setContext on the Sentry appender elsewhere -- does anyone know what this is needed for and how to do it in sentry-logback?

**Comment by lmath** [comment]
@pvighi, I see you replaced raven-logback with sentry-logback in members-data-api, so if you're feeling charitable, you might take a look at this PR 😄

File: app/monitoring/SentryLogging.scala [hunk 2]
```diff
@@ -1,31 +1,28 @@
...
+import scala.collection.JavaConverters._
+import scala.util.{Failure, Success, Try}
+
+object SentryLogging {
+  def init(config: Configuration) {
+    config.sentryDsn match {
+      case None => SafeLogger.warn("No Sentry logging configured (OK for dev)")
+      case Some(sentryDSN) =>
+        SafeLogger.info(s"Initialising Sentry logging with $sentryDSN")
+        Try {
```
Comment (jacobwinch) [inline app/monitoring/SentryLogging.scala]: 👍for wrapping this in a Try

File: app/monitoring/SentryLogging.scala [hunk 3]
```diff
@@ -1,31 +1,28 @@
...
+import config.Configuration
+import io.sentry.Sentry
 
-class SentryLogging(dsnConfig: String, stage: Stage) {
-  val dsn = new Dsn(dsnConfig)
-  SafeLogger.info(s"Initialising Sentry logging for ${dsn.getHost}")
-  val tags = Map("stage" -> stage.toString) ++ BuildInfo.toMap
-  val tagsString = tags.map { case (k, v) => s"$k:$v" }.mkString(",")
-  val sentryAppender = new SentryAppender(RavenFactory.ravenInstance(dsn)) {
-    addFilter(SentryFilters.errorLevelFilter)
```
Comment (jacobwinch) [inline app/monitoring/SentryLogging.scala]: I think this means you can delete: https://github.com/guardian/support-frontend/blob/16b54b1f7d02481f387ce09b38c68628cafb800a/app/monitoring/Logging.scala#L53-L61?

File: conf/logback.xml [hunk 4]
```diff
@@ -23,8 +23,18 @@
     </encoder>
 </appender>
 
+<appender name="Sentry" class="io.sentry.logback.SentryAppender">
+    <filter class="ch.qos.logback.classic.filter.ThresholdFilter">
+        <level>ERROR</level>
+    </filter>
+    <filter class="monitoring.PiiFilter">
```
Comment (jacobwinch) [inline conf/logback.xml]: Might be worth adding a comment to this code...

https://github.com/guardian/support-frontend/blob/16b54b1f7d02481f387ce09b38c68628cafb800a/app/monitoring/Logging.scala#L10-L13

... to explain that the filter is referenced in logback.xml.

File: app/monitoring/SentryLogging.scala (hunk 3 above)
Comment (lmath) [reply app/monitoring/SentryLogging.scala]: Ah, yes! Well spotted.

File: conf/logback.xml (hunk 4 above)
Comment (lmath) [reply conf/logback.xml]: That would be a useful comment. Will do.

Comment (jacobwinch) [reply conf/logback.xml]: Alernatively you could define the filter itself here in logback.xml i.e. the opposite filter to 

https://github.com/guardian/support-frontend/blob/master/conf/logback.xml#L7-L12

**Pull request merged**